sigle/src/types/index.ts

```diff
@@ -103,3 +103,26 @@ export interface SettingsFile {
    */
   siteTwitterHandle?: string;
 }
+
+export interface AnalyticsHistoricalData {
```

Comment:
As these types are only used by the analytics, the types should probably stay within the stats module

Reply:
would it be worth putting re-used types like `StatsData` in a separate types file within the stats module?

Reply:
Sure 👍